src/Model/Filter/Like.php

```diff
@@ -29,6 +30,11 @@ public function process()
             return;
         }
 
+        $driver = $this->query()->connection()->driver();
+        if (!$driver instanceof Postgres) {
```

Comment:
Maybe we should have a configurable map for the comparisons instead of doing this, that way we might accurately support all datastores?